src/test/java/codeu/model/store/basic/UserStoreTest.java

```diff
@@ -74,6 +74,18 @@ public void testGetUser_byId_notFound() {
     Assert.assertNull(resultUser);
   }
 
+  @Test
+  public void testAddUserByName() {
+    userStore.addUser("test username1", "Password1", false);
```

Comment:
small comment can go on 'false' here too

Reply:
Thank you, fixed!